### DISCUSSION TITLE
Standardize the media sequence of The Watchtower

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#1917] [Ideas] [answered]
### Is your idea related to a problem?

It's not a problem.

### Describe your idea

Would it be possible to **standardize** the numbering of the Watchtower media to the same format as the midweek meeting media?

Midweek media numbering is in this format 00-00-00 (6 fields / digits)

Currently W media are only 4 digits/fields (00-00)

![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/f4c65f34-9699-49ad-b1f5-11252cac3371)

### Describe alternatives you've considered

It's just a suggestion to standardize, in case it doesn't turn out to be a big job to do for this change.

**Comment by mtdvlpr** [comment, marked as answer]
Resolved as of `v24.10.6`
